pyQuARC/__init__.py

```diff
@@ -15,5 +17,8 @@
 with open(f"{ABS_PATH}/version.txt") as version_file:
     __version__ = version_file.read().strip()
 
-def get_version():
+def version():
     return __version__
+
+def supported_formats():
```

Comment:
Can we add a short description about what the formats are?

Reply:
maybe just a link to some other documentation

Reply:
done